src/components/course/CourseReview.jsx

```diff
@@ -110,7 +110,7 @@ const CourseReview = () => {
             onClick={() => { setShowInfoContent(REVIEW_SECTION_CONTENT.DEMAND_AND_GROWTH); }}
             className={`number-color mb-0 ${ showInfoContent === REVIEW_SECTION_CONTENT.DEMAND_AND_GROWTH && 'text-underline'}`}
           >
-            {courseReviews.totalEnrollments}
+            {courseReviews?.totalEnrollments.toLocaleString('en-US')}
```

Comment:
[nit/inform] The `?.` here isn't necessary. Missing `courseReviews` is already handled above. This code path can't happen if `courseReviews` is missing due to the above:

```jsx
if (!courseReviews) {
  return null;
}
```